### DISCUSSION TITLE
Welcome to edcell.live Discussions!

### DISCUSSION THREAD

**Discussion by RahulKumarSingh123** [RahulKumarSingh123/edcell.live#7] [Announcements]
Welcome 
Can anyone suggest some options for background of the webpage?

**Comment by LavkushSolanki** [comment]
You can add black background this this whole website it will look good very good as per the theme you have set up

**Reply by RahulKumarSingh123** [reply]
What is this  can you explain?

**Reply by LavkushSolanki** [reply]
You can make background-color:"black" In the css file of index.css

**Reply by RahulKumarSingh123** [reply]
Ok you do it and make pull request I will add

**Comment by LavkushSolanki** [comment]
Hello I am also contributing to this project
